     public static void showTextSelectionDialog(Context ctx, String text) {
         MaterialAlertDialogBuilder selectTextDialogBuilder = new MaterialAlertDialogBuilder(ctx);
@@ -21,7 +44,7 @@
         selectTextDialogBuilder.setView(rootView);
 
         HtmlTextView htmlTextView = rootView.findViewById(R.id.select_text_htmltextview);
-        htmlTextView.setHtml(text);
+        htmlTextView.setHtml(untruncateLinks(text));
 
         htmlTextView.setOnClickATagListener(new OnClickATagListener() {
             @Override"